packages/server/api/controllers/movies.controller.js

```diff
@@ -24,6 +24,31 @@ const getMovieByID = async (id) => {
   // }
 };
 
+const getCrewOfMovieByID = async (id) => {
+  if (!id) {
+    throw new HttpError('ID should be a number', 400);
+  }
+  try {
+    const crew = await knex
+      .select(
+        'crew_members.id',
+        'full_name',
+        'birth_date',
+        'image_location',
+        'role',
+      )
+      .from('movie_crew')
+      .join('crew_members', 'movie_crew.crew_member_id', '=', 'crew_members.id')
+      .where('movie_id', id);
+    if (!crew || crew.length === 0) {
+      throw new Error(`No crew found for this movie`, 404);
```

Comment:
Throw error and catch in same block can lead to both throwing exception.
Have you tried to test this with an id that does not exist and see you can get expected error?

Reply:
you are correct, i will send a message instead of throw error

Reply:
Maybe moving this error handling to router would be optimal.
You can refer to this PR https://github.com/HackYourFuture-CPH/CINEMANIA/pull/50/files#diff-f8f4871279607cea2f43141bd7d3600a2ead0fa38bb516e7d5be1eee4c308359

Reply:
i moved the error handling to router file and i will take the same method next time 👍

Reply:
Looks good. 
Could you move crew into a separate controller and router?

Reply:
i make a new separate controller and router for the cast, and waiting for the review 🙏